packages/flutter/test/rendering/debug_test.dart

```diff
@@ -208,4 +208,30 @@ void main() {
     expect(error, isNull);
     debugPaintSizeEnabled = false;
   });
+
+  test('debugDisableOpacity keeps things in the right spot', () {
```

Comment:
Without this patch, this test can pass also, right?

Reply:
The container gets rendered in the wrong location without this patch

Reply:
Err, sorry - without this change, there'd be no `OpacityLayer`, the `PictureLayer` would be the `firstChild` of `rootLayer`.

Reply:
It passes with the latest master branch.

Reply:
Ahh you are right. This would need to test that the add to scene does the right thing. But the golden test does that without getting into such implementation details. I'll remove this test.

Reply:
It isn't bad to remain this ^^

Reply:
I agree, it's not bad to keep this test, even if it passes without this change and has coverage in a golden file test. I think it's never bad to have another test. :)

Reply:
Left it in :)